src/test/java/Task30/Locators.java

```diff
@@ -0,0 +1,14 @@
+package Task30;
+
+import org.openqa.selenium.By;
+
+
+public class Locators {
+
+    public static final By LOGIN_BUTTON = By.xpath("//a[contains(@class,'Button2_view_default Button2_size_m')]");
+    public static final By USERNAME_FIELD = By.id("passp-field-login");
+    public static final By PASSWORD_FIELD = By.xpath("//input[@id ='passp-field-passwd']");
+    public static final By SUBMIT_BUTTON = By.xpath("//button[contains(@class,'submit')]");
```

Comment:
try to use different By locators. Ex.:
public static final By SUBMIT_BUTTON = By.**cssSelector**("[type='submit']");
public static final By PASSWORD_FIELD = By.**className**("Textinput-Control");
public static final By LOGOUT_BUTTON = By.**linkText**("Log out");